colosseum/declaration.py

```diff
@@ -45,10 +48,20 @@ def deleter(self):
 
 def validated_property(name, choices, initial):
     "Define a simple CSS property attribute."
-    initial = choices.validate(initial)
+    try:
+        initial = choices.validate(initial)
+    except ValueError:
+        # The initial value might be a OtherProperty or Custom class with a value method
```

Comment:
In which case, we should validate that, right? We can validate that the value of initial *has* a value method...

Reply:
Fixed!